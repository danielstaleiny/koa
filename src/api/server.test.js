const mongoose = require('../../lib/db/mongoose')
const request = require('supertest')
const app = require('../server')

const User = require('../model/user')

beforeAll(async () => {
    // await User.remove({}).exec();
})
test('root route', async () => {
    const response2 = await request(app.callback())
        .post('/users')
        .send({
            id: 5,
            text: 'anon'
        })

    expect(response2.body).toMatchSnapshot()
    const response = await request(app.callback()).get('/users/5')
    expect(response.body).toMatchSnapshot()
    console.log(response.body)
})

afterAll(async () => {
    await User.remove({}).exec()
    mongoose.mongoose.connection.close()
})
