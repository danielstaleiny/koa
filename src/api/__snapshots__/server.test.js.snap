// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root route 1`] = `
Object {
  "id": "5",
  "text": "anon",
}
`;

exports[`root route 2`] = `
Object {
  "id": "5",
  "text": "anon",
}
`;
